modules/runner/src/integrationTest/java/org/apache/ignite/internal/sqllogic/SqlScriptRunner.java

```diff
@@ -244,78 +289,58 @@ boolean ready() throws IOException {
             return buffReader.ready();
         }
 
-        String positionDescription() {
-            return '(' + fileName + ':' + lineNum + ')';
+        ScriptPosition scriptPosition() {
+            return new ScriptPosition(fileName, lineNum);
         }
 
         @Override
         public void close() throws Exception {
             buffReader.close();
         }
 
+        @Nullable
         private Command nextCommand() {
```

Comment:
```suggestion
        private @Nullable Command nextCommand() {
```

Reply:
> Les't fix styles and extract static Command classes from SqlScriptRunner to the top-level classes.

Done + added javadocs and some test cases to _runner/self.test

Reply:
Fixed

Reply:
fixed